cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -569,4 +580,22 @@ and then restart CDAP.
 #. Restart the CDAP processes::
 
      for i in `ls /etc/init.d/ | grep cdap` ; do sudo service $i start ; done
+     
+If you are upgrading a secure Hadoop cluster, see the notes above about
+:ref:`configuring secure Hadoop <install-secure-hadoop>`. 
+
+.. _install-troubleshooting:
+
+Troubleshooting
+---------------------------------
+
+- If you have YARN configured to use LinuxContainerExecutor (see the setting for
+  ``yarn.nodemanager.container-executor.class``), the ``cdap`` user needs to be present on
```

Comment:
This "cdap" mention and the one below should technically be "<cdap-principal>" as well.